notebook/static/notebook/js/notebook.js

```diff
@@ -65,6 +65,7 @@ define(function (require) {
         this.ws_url = options.ws_url;
         this._session_starting = false;
         this.last_modified = null;
+        this.line_numbers = false;
```

Comment:
I think I would make that private with a leading underscore, so that we can change the implementation easily.